packages/react/legacy/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "private": true,
+  "name": "@aws-amplify/ui-react-legacy",
+  "main": "../dist/legacy.js",
+  "module": "../dist/esm/legacy.js",
+  "types": "../dist/legacy.d.ts"
```

Comment:
This file is similar to https://github.com/aws-amplify/amplify-js/blob/82544a6c8e0c0ae30f988b066e7e250f49862f64/packages/amplify-ui-components/loader/package.json in that helps TypeScript find `@aws-amplify/ui-react/legacy`'s types, since `types` can only be a string, not an array.

But this also helps for CJS bundlers that don't support the `exports` field.